tests/test_travis_mf_multiplePS/worker_mf.cpp

```diff
@@ -0,0 +1,63 @@
+#include <unistd.h>
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+#include <thread>
+#include <vector>
+
+#include <Configuration.h>
+#include <InputReader.h>
+#include <PSSparseServerInterface.h>
+#include <SparseMFModel.h>
+#include <Tasks.h>
+#include "SGD.h"
+#include "Serializers.h"
+#include "Utils.h"
+
+using namespace cirrus;
+
+cirrus::Configuration config = cirrus::Configuration("configs/jester.cfg");
+
+int main() {
+  InputReader input;
+  int nusers, njokes;
+  SparseDataset train_dataset = input.read_jester_ratings(
+      "tests/test_data/jester_train.csv", &nusers, &njokes);
+  train_dataset.check();
+  train_dataset.print_info();
+  int nfactors = 10;
+  int batch_size = 200;
+
+  SparseMFModel model(nusers, njokes, nfactors);
+
+  std::vector<std::string> ps_ips;
```

Comment:
I think you can do, ```ps_ips{"127.0.0.1", "127.0.0.1"};``` Same below.

Reply:
Done

Reply:
I actually used ps_ips("127.0.0.1", 2), which is the same thing.